packages/noya-designsystem/src/components/GridView.tsx

```diff
@@ -64,37 +64,74 @@ const ItemDescription = styled.span(({ theme }) => ({
   textOverflow: 'ellipsis',
 }));
 
+type MenuItemType = 'delete' | 'copy';
```

Comment:
(See other comment) So we shouldn't have any specific item values like "delete" or "copy" in this file